uwblueprint/supportive-housing
# Employee table edit and delete

## GitHub Issue link

[Employee Table Edit Delete](https://github.com/uwblueprint/supportive-housing/issues/95)



## Implementation description
* 



## Steps to test
1. Add an employee
2. Ensure that employee can be edited and all fields required cannot be left blank
3. Ensure that the row data updates after submitting
4. Ensure cancelling works
5. Ensure that you can only delete that employee, the modal header/message are correct, and cancelling works
6. Sign up with an invited employee (to be active)
7. Ensure that you can only deactivate that employee, the modal header/message are correct, and cancelling works
8. Ensure that after deactivating, you can only activate that employee, the modal header/message are correct, and cancelling works


## What should reviewers focus on?
* 


## Checklist
